_test/content_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
+
+describe "Content" do
+
+  BannedCharacters = [
+    "\u201C", # U+201C “
+    "\u201D", # U+201D ”
+    "\u201E", # U+201E „
+    "\u2018", # U+2018 ‘
+    "\u2019", # U+2019 ’
+    "\u201A", # U+201A ‚
+    "\u201B", # U+201B ‛
+  ]
```

Comment:
I found my way here via the dnsimple blog, curious about the kinds of tests that might apply to a documentation repo.

Why are these specific characters banned?